lib/src/dom/config/configure.dart

```diff
@@ -29,16 +27,16 @@ export 'package:react_testing_library/src/dom/config/types.dart' show JsConfig;
 ///
 /// > See: <https://testing-library.com/docs/dom-testing-library/api-configuration/>
 void configure({
-  String testIdAttribute,
-  int asyncUtilTimeout,
-  bool computedStyleSupportsPseudoElements,
-  bool defaultHidden,
-  bool showOriginalStackTrace,
-  bool throwSuggestions,
-  TestingLibraryElementError Function(Object message, Element container) getElementError,
+  String? testIdAttribute,
+  int? asyncUtilTimeout,
+  bool? computedStyleSupportsPseudoElements,
+  bool? defaultHidden,
+  bool? showOriginalStackTrace,
+  bool? throwSuggestions,
+  TestingLibraryElementError Function(Object? message, Element container)? getElementError,
```

Comment:
That's because it's [non-nullable in TS](https://github.com/testing-library/dom-testing-library/blob/a7b725257e61370675ee043813003d36fae23e9d/types/config.d.ts#L19). A lot of `container` related arguments are nullable because the default `screen.container` is set to `document.body` which is nullable. I talked to Greg about it though and he said it's safe to just enforce that `document.body` is non-null by the time it's used so then I can make all `container` args non-nullable instead so I'll update those

Reply:
Though now looking at that TS type, I'm not sure why `message` is typed as `Object?` instead of `String?` - do you happen to remember?

Reply:
I have no idea, honestly